src/components/DateSelector.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div>
+    <va-date-input
+    label="due date"
+    v-model="value"
+    :format="formatDate"
+    :parse="parseFn"
+    placeholder="yyyy-mm-dd"
+    clearable
+    manual-input
+  />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, watch, PropType, onMounted } from "vue";
+import { Task } from "@/types/asana";
+import { format, isValid, parse } from "date-fns";
+import store from "@/store";
+
+export default defineComponent ({
+  props: {
+    task: {
+      type: Object as PropType<Task>,
```

Comment:
Probably should just be a date, not the task (in case we want to use for other dates)